mateweather/mateweather-dialog.c

```diff
@@ -502,6 +502,45 @@ static PangoFontDescription* get_system_monospace_font(void)
     return desc;
 }
 
+static void
+override_widget_font (GtkWidget            *widget,
+		      PangoFontDescription *font)
+{
+	GtkCssProvider *provider;
+	gchar          *css;
+	gchar          *family;
+	gchar          *weight;
+	gchar          *style;
+	gchar          *size;
+
+	family = g_strdup_printf ("font-family: %s;", pango_font_description_get_family (font));
+
+	weight = g_strdup_printf ("font-weight: %d;", pango_font_description_get_weight (font));
+
+	if (pango_font_description_get_style (font) == PANGO_STYLE_NORMAL)
+		style = g_strdup_printf ("font-style: normal;");
+	else if (pango_font_description_get_style (font) == PANGO_STYLE_ITALIC)
+		style = g_strdup_printf ("font-style: italic;");
+	else
+		style = g_strdup_printf ("font-style: oblique;");
```

Comment:
For `style` you can use `const gchar *`, no? No need to copy/allocate extra memory...